File: String/Anagram1/Anagram.cc
class Anagram {
	public:
		bool anagram(string s, string t) {
			if(s.empty() || t.empty())
				return false;
			
			if(s.size() != s.siz())
				return false;
			
			sort(s.begin(), s.end());
			sort(t.begin(), t.begin());
			
			if(s == t)
				return true;
			else
				return false;
		}
};

File: String/Anagram1/Anagram.java
public class Anagram {
	public boolean anagram(String s, String t) {
		if(s == null || t == null)
			return false;
		
		if(s.length() != t.length())
			return false;
		
		char[] sCharArray = s.toCharArray();
		char[] tCharArray = t.toCharArray();
		
		Array.sort(sCharArray);
		Array.sort(tCharArray);
		
		for(int i = 0; i < s.length(); i++) {
			if(sCharArray[i] != toCharArray[i])
				return false;
		}
		
		return true;
 	}
}